src/resources/2014/ConspiracyIsland.ts

```diff
@@ -0,0 +1,286 @@
+import { canInteract, Item, Location, runChoice, visitUrl } from "kolmafia";
+import { get } from "../../property";
+import { $item, $location } from "../../template-string";
+
+const radio = "place.php?whichplace=airport_spooky&action=airport2_radio";
+const omegapanel =
+  "place.php?whichplace=airport_spooky_bunker&action=si_controlpanel";
+
+export function have(): boolean {
+  return get("spookyAirportAlways");
+}
+
+class conspiracyQuestData {
```

Comment:
It is unclear to me why this is a Class

Reply:
are you querying the necessity of it at all, or just why it is a class instead of [other thing]? If the latter, because I dont know what else to make it (i tried to make it a "record"(?) but could not get that to work.